package.json

```diff
@@ -85,11 +86,16 @@
     "react-toastify": "^8.2.0",
     "react-tooltip": "^4.2.21",
     "react-youtube": "^7.12.0",
+    "@react-spring/web": "^9.7.3",
     "socket.io-client": "^4.5.4",
     "sortablejs": "^1.12.0",
     "swiper": "^8.4.4",
     "time-ago": "^0.2.1",
-    "tinycolor2": "^1.4.2"
+    "tinycolor2": "^1.4.2",
+    "bootstrap": "^5.3.0",
+    "d3": "^7.8.4",
+    "html-react-parser": "^4.2.0",
+    "react-bootstrap": "^2.8.0"
```

Comment:
We don't use react-bootstap and bootstrap currently in the project, so these dependencies should be removed and the code should be replaced with normal html and css classes. Also, we are currently stuck at Bootstrap 3 and can't use Bootstrap 5.

Reply:
Fixed